BitmarkRegistry/Models/LatestOffsetR.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  LatestOffsetR.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 7/13/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+class LatestOffsetR: Object {
```

Comment:
This one beyond the scope of db, since it only has one field. Should we save in `Userdefault` instead?

Reply:
Ok, I will try to update it. Thanks